heat/core/tensor.py

```diff
@@ -271,12 +293,84 @@ def max(self, axis=None, out=None):
 
         return operations.max(self, axis, out)
 
-    def mean(self, axis):
-        # TODO: document me
-        # TODO: test me
-        # TODO: sanitize input
-        # TODO: make me more numpy API complete
-        return self.sum(axis) / self.shape[axis]
+    def mean(self, dimen=None, all_procs=False):
```

Comment:
Should expose the numpy.mean() interface (see here: https://docs.scipy.org/doc/numpy/reference/generated/numpy.mean.html)